packages/velo-external-db-core/lib/router.js

```diff
@@ -9,10 +9,12 @@ const { secretKeyAuthMiddleware } = require('./web/auth-middleware')
 const { authRoleMiddleware } = require('./web/auth-role-middleware')
 const { unless, includes } = require('./web/middleware-support')
 const path = require('path')
-const { hooksForAction, Operations, payloadFor, Actions, requestContextFor } = require('./hooks_utils')
-const { FIND, INSERT, BULK_INSERT, UPDATE, BULK_UPDATE, REMOVE, BULK_REMOVE, AGGREGATE, COUNT, GET } = Operations
+const { hooksForAction: dataHooksForAction, Operations: dataOperations, payloadFor: dataPayloadFor, Actions: DataActions, requestContextFor } = require('./data_hooks_utils')
+const { hooksForAction: schemaHooksForAction, Operations: SchemaOperations, payloadFor: schemaPayloadFor, Actions: SchemaActions } = require ('./schema_hooks_utils')
```

Comment:
I would drop the multiple renames, and do the following:
Don't deconstruct the imported members from `data_hooks_utils` and `schema_hooks_utils`, and just do something like `const dataHooksUtils = require("./data-hooks-utils")`, and then for example you can do something like `dataHooksUtils.Operations`/`schemaHooksUtils.Operations`.